mon-pix/app/components/navbar-desktop-header.js

```diff
@@ -22,4 +22,8 @@ export default class NavbarDesktopHeader extends Component {
       { name: this.intl.t('navigation.not-logged.sign-up'), link: 'inscription', class: 'navbar-menu-signup-link' },
     ];
   }
+
+  get _isFromExternalPlatform() {
+    return this.session.get('data.externalUser');
+  }
```

Comment:
Cela fait plusieurs PR que l'on utiliser le terme "external user". Bien sûr, il y a bien un enregistrement dans la table users, mais il ne sert qu'à identifier, et pas à authentifier. L'origine de cet utilisateur est externe  ( = il n'a pas créé de compte Pix, bien qu'en fait il a pu en créer un avec un email, puis en se réconciliant ajouter une méthode de connexion). 

Bref, je balance entre la cohérence (externalUser) et le côté explicite (UserFromExternalIDP), plutôt que d'utiliser platform qui est un peu vague (aussi vague que `environment`, `application`, `source`..)
```suggestion
  get _isUserExternal() {
    return this.session.get('data.externalUser');
  }
```

Reply:
Done.

Reply:
@octo-topi En fait ce n'est pas l'utilisateur qui est externe (tous les utilisateurs sont internes, en définitive…) mais la méthode d'authentification qu'il a utilisée !